src/ProjectNeon/Assets/Battle/Enemies/EnemyVisualizer.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyVisualizer : MonoBehaviour
+{
+
+    [SerializeField] private EnemyArea enemyArea;
+    [SerializeField] private List<GameObject> enemies;
+
+    /**
+     * @todo #29:15min Draw enemy characters on battle screen. Enemy positioning must be relative to 

```

Comment:
Do we need this puzzle? 

Can we do the spacing in this ticket?

Or, should we reduce the scope on this one, since this is one of the main interesting pieces of behavior here...